Microsoft.Research/ClousotMain/RegressionOutput.cs

```diff
@@ -333,7 +318,8 @@ protected MethodRegressionOutcomes(Set<string> outcomes) : base(outcomes)
         {
           var outcome = (ProofOutcome)(byte)GetNamedArgOrDefault<int, Local, Parameter, Method, Field, Property, Event, Type, Attribute, Assembly>(mdDecoder, "Outcome", attr);
 
-          string msg = (string)mdDecoder.NamedArgument("Message", attr);
+          // at some places we have "this.x" instead of "x", both should be treated equal.
```

Comment:
What about situation where "this." is a tail of some other name for example "this.watchthis." ?